<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrugSaleManageSystemMainWidget</class>
 <widget class="QWidget" name="DrugSaleManageSystemMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仓库管理系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="pixmap">
        <pixmap resource="res.qrc">:/pic/title.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>串口列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serialPortList"/>
     </item>
     <item>
      <widget class="QPushButton" name="refreshSerialPortList">
       <property name="text">
        <string>刷新串口列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openCloseBtn">
       <property name="text">
        <string>Closed</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/pic/shut_down_grayed.png</normaloff>:/pic/shut_down_grayed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>仓库分区：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="refreshBtn">
          <property name="text">
           <string>重新读卡</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="opListTable">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="PageSaleOutWidget" name="saleOutPage">
         <attribute name="title">
          <string>出库清单</string>
         </attribute>
        </widget>
        <widget class="PageBuyInWidget" name="buyInPage">
         <attribute name="title">
          <string>入库清单</string>
         </attribute>
        </widget>
        <widget class="PageSaleRecordWidget" name="recordPage">
         <attribute name="title">
          <string>出入库记录</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PageSaleRecordWidget</class>
   <extends>QWidget</extends>
   <header>PageSaleRecordWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageSaleOutWidget</class>
   <extends>QWidget</extends>
   <header>PageSaleOutWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageBuyInWidget</class>
   <extends>QWidget</extends>
   <header>PageBuyInWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
